include/IDriver.h

```diff
@@ -227,7 +228,7 @@ class IDriver : public virtual core::IReferenceCounted, public IVideoCapabilityR
 		{
 			auto retval = createDeviceLocalGPUBufferOnDedMem(size);
 
-			updateBufferRangeViaStagingBuffer(retval.get(), 0u, size, data);
+			//updateBufferRangeViaStagingBuffer(retval.get(), 0u, size, data);
```

Comment:
this one might be tricky... should it:
- be a ILogicalDevice method that takes a queue pointer?
- a Queue method?

Probably the first